Moblin/Various/Settings.swift

```diff
@@ -1125,7 +1125,10 @@ class SettingsRtmpServerStream: Codable, Identifiable {
     var name: String = "My stream"
     var streamKey: String = ""
     var latency: Int32? = defaultRtmpLatency
+    var manualFps: Bool? = false
     var fps: Double? = 0
+    var autoSelectRtmpScene: Bool? = false
+    var autoSelectRtmpMic: Bool? = false
```

Comment:
I prefer not adding the auto selection settings now. Maybe ok with mic, but not scene as there is no automatic scene selection anywhere else in Moblin. Can maybe add leter, but let's not do it now.

Reply:
I can't quite understand that. Because from the user's perspective, if I connect an RTMP publisher, I will also want to stream through it. Every click that can be saved contributes to usability. especially when the decision is made to the user. but you are the boss :) I have now taken it out.

Reply:
thanks, I'm sure we will revisit it later